Stripe/PublicHeaders/STPAPIClient.h

```diff
@@ -360,6 +360,27 @@ static NSString *const STPSDKVersion = @"15.0.0";
 
 @end
 
+
+#pragma mark Payment Methods
+
+/**
+ STPAPIClient extensions for working with PaymentMethod objects.
+ */
+@interface STPAPIClient (PaymentMethods)
+
+/**
+ Creates a PaymentMethod object with the provided params object.
+ 
+ @see https://site-admin.stripe.com/docs/api/payment_methods/create
```

Comment:
Let's remove the `site-admin` part of this and just use the normal "stripe.com" url